ios/RNCallKeep/RNCallKeep.m

```diff
@@ -404,6 +412,20 @@ + (void)reportNewIncomingCall:(NSString *)uuidString
                   fromPushKit:(BOOL)fromPushKit
                       payload:(NSDictionary * _Nullable)payload
         withCompletionHandler:(void (^_Nullable)(void))completion
+{
+    [RNCallKeep reportNewIncomingCall:uuidString handle:handle handleType:handleType hasVideo:hasVideo localizedCallerName:localizedCallerName fromPushKit:fromPushKit payload:payload withCompletionHandler:nil resolver:nil rejecter:nil];
+}
+
++ (void)reportNewIncomingCall:(NSString *)uuidString
+                       handle:(NSString *)handle
+                   handleType:(NSString *)handleType
+                     hasVideo:(BOOL)hasVideo
+          localizedCallerName:(NSString * _Nullable)localizedCallerName
+                  fromPushKit:(BOOL)fromPushKit
+                      payload:(NSDictionary * _Nullable)payload
+        withCompletionHandler:(void (^_Nullable)(void))completion
```

Comment:
Skal vi ikke bare undlade `withCompletionHandler`? Når vi har en resolver og rejecter, behøver vi vel ikke yderligere en callback?

Reply:
withCompletionHandler er noget specifikt til callkit. Man får et input "withCompletionHandler:(void(^)(void)) i AppDelegate.m/didReceiveIncomingPushWithPayload, som skal blive kaldt i tilfældet hvor et opkald er blevet startet.
I henhold til react koden, så bliver denne callback ikke brugt.

Reply:
Got it. 👍